source/docs/software/advanced-controls/trajectories/troubleshooting.rst

```diff
@@ -144,6 +138,19 @@ If your feedforwards are bad then the P controllers for each side of the robot w
         disabledRamsete,
         ...
     );
+    
+   .. code-tab:: c++
+
+    frc::RamseteController m_disabledRamsete;
+    disabledRamsete.SetEnabled(false);
+
+    // Be sure to pass your new disabledRamsete variable
+    frc2::RamseteCommand ramseteCommand(
+      exampleTrajectory,
+      [this]() { return m_drive.GetPose(); },
+      disabledRamsete,
+      ...
```

Comment:
How come we do this in C++ but not java

Reply:
We do what in C++ and not Java?
Assuming you're referring to the lambda: in Java it's a method ref but C++ doesn't have method refs so it's a lambda.

Reply:
Oh nvm, am bad